docs/using_deis/index.rst

```diff
@@ -19,5 +19,6 @@ Using Deis
     using-buildpacks
     using-dockerfiles
     using-docker-images
+    using-websockets
```

Comment:
This should be under "Managing Deis". "Using Deis" is reserved for using the platform.